tests/tests_integration/test_api/test_files.py

```diff
@@ -30,6 +30,10 @@ def test_files():
 
 
 class TestFilesAPI:
+    def test_create(self):
+        file_metadata, upload_url = COGNITE_CLIENT.files.create(name="mytestfile")
```

Comment:
Maybe just add an `assert False == file_metadata.uploaded` before deleting the file?